db/migrate/20210628213217_add_post_edits_count_to_user_stats.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddPostEditsCountToUserStats < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_stats, :post_edits_count, :integer, default: 0, null: false
+
+    execute <<~SQL
+      UPDATE user_stats us
+      SET post_edits_count = editor.edits_count
+      FROM (
+        SELECT COUNT(editor.id) AS edits_count, editor.id AS id
+        FROM post_revisions pr JOIN users editor ON editor.id = pr.user_id
+        GROUP BY editor.id
+      ) editor
+      WHERE editor.id = us.user_id;
```

Comment:
A recent performance problem we encountered has made me wary of queries that backfill records in one go. Can we update this query such that it backfills in batches instead? You can have a look at https://github.com/discourse/discourse/commit/14a0247301510a5703d1d4b7f62141c13935cc97 for the common pattern which we use.